src/Sql/dbo/Stored Procedures/Organization_DeleteById.sql

```diff
@@ -62,10 +62,74 @@ BEGIN
     EXEC [dbo].[OrganizationSponsorship_OrganizationDeleted] @Id
 
     DELETE
+    FROM
+        [dbo].[OrganizationPasswordManager]
+    WHERE
+        [OrganizationId] = @Id
+
+    DELETE
+    FROM
+        [dbo].[OrganizationSecretsManager]
+    WHERE
+        [OrganizationId] = @Id
+
+   DELETE
     FROM
         [dbo].[Organization]
     WHERE
         [Id] = @Id
 
     COMMIT TRANSACTION Organization_DeleteById
 END
+GO
+
+IF EXISTS (
+ SELECT *
+    FROM INFORMATION_SCHEMA.COLUMNS
+    WHERE COLUMN_NAME = 'Plan' AND
+        DATA_TYPE = 'NVARCHAR' AND
+        TABLE_NAME = 'Organization' AND 
+        IS_NULLABLE = 'NO')
+BEGIN
+    ALTER TABLE [dbo].[Organization]
+    ALTER COLUMN [Plan] NVARCHAR(50) NULL
+END
+GO
+
+IF EXISTS (
+    SELECT *
+    FROM INFORMATION_SCHEMA.COLUMNS
+    WHERE COLUMN_NAME = 'PlanType' AND
+        DATA_TYPE = 'tinyint' AND
+        TABLE_NAME = 'Organization' AND 
+        IS_NULLABLE = 'NO')
+BEGIN
+    ALTER TABLE [dbo].[Organization]
+    ALTER COLUMN PlanType tinyint NULL
+END
+GO
+
+IF EXISTS (
+    SELECT *
+    FROM INFORMATION_SCHEMA.COLUMNS
+    WHERE COLUMN_NAME = 'UseTotp' AND
+        DATA_TYPE = 'bit' AND
+        TABLE_NAME = 'Organization' AND
```

Comment:
I don't think these belong in a sproc definition (couldn't scroll to select all)

Reply:
Thanks!